connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java

```diff
@@ -296,6 +297,56 @@ public HeaderConverter newHeaderConverter(AbstractConfig config, String classPro
         return plugin;
     }
 
+
+    /**
+     * If the given configurations defines a {@link ConnectRestExtension} using the provided
+     * Class Names, return a list of new configured instances.
+     *
+     * @param klassNames         the list of Class Names of {@link ConnectRestExtension} that
+     *                           needs to instantiated and configured
+     * @param config             the configuration containing the {@link org.apache.kafka.connect.runtime.Worker}'s configuration;
+     *                           may not be null
+     * @return the instantiated and configured list of  {@link ConnectRestExtension}; EMPTY if
+     * the {@param klassNames} is NULL or EMPTY
+     * @throws ConnectException if the {@link ConnectRestExtension} implementation class could not be found
+     */
+    public List<ConnectRestExtension> newConnectRestExtensions(List<String> klassNames,
+                                                               AbstractConfig config) {
```

Comment:
This method does not have much logic that is specific to `ConnectRestExtension`. Have you thought about making this more general-purpose to load any extensions (other than when a more specific method is available)? Doing that might make it easier to add future extensions.